### DISCUSSION TITLE
One strange thing in Helix buffer and splits.

### DISCUSSION THREAD

**Discussion by joaocarvalhoopen** [helix-editor/helix#4637] [General] [answered]
Hello,

If I have a  split it is like a window split, right? 
And I should have a way to list all the buffers that I have on that split(window), then I should have buffers attributed for each split that I could go next and backwords between buffers.

Mainly:
1. Each split has a collection of buffers.

2. Buffer views should be possible to copy between splits/window splits.

3. It should be possible to list all buffers inside a split. Like seeing all open tabs.

4. It should be possible to jump between buffer inside a split in a random way, and not only in a bi-directional list way.

I don't have experience with VIM, what I'm saying makes sense or is completely dumme?

What's the advantage of being like it is currently?

Best regards,
João

**Comment by the-mikedavis** [comment, marked as answer]
The buffer picker `<space>b` lists and picks from the open buffers.

Like in Vim, buffers are available from all splits so it isn't necessary to copy buffers between splits.

Isolating buffers could be part of https://github.com/helix-editor/helix/issues/2295 but I don't see any reason to isolate buffers between splits - it would just make it harder to switch buffers.

**Comment by joaocarvalhoopen** [comment]
Thank you for the reponse.

I have tested and the buffer picker was what I was looking for, a list of all the open buffers,
It even had the bonus of the previewer.
There is only one thing that I notice, is that you can't really close opened buffers without quiting the program.
I tryed to open a file_A , open a vsplit then open in it a file_B then I closed the vsplit buffer and it cosed the Vsplit view, but the buffer still appears in the list of opened buffer in the buffer picker. 
I think that is not a bad thing per si, just not what i expected when I saw a close buffer option in the "Space ?" option menu.

Best regards,
João

**Reply by the-mikedavis** [reply]
Closing splits doesn't close buffers: you can use the `:buffer-close*` commands instead.
